core/kernel/printf_spec.rb

```diff
@@ -31,6 +31,13 @@
     object.should_receive(:write).with("string")
     Kernel.printf(object, "%s", "string")
   end
+
+  it "calls to_str on the second argument when it is not a string" do
```

Comment:
**suggestion:**
```suggestion
  it "calls #to_str on the second argument when it is not a String" do
```

Reply:
Here I reused the same message too. I'm totally fine with updating them, but I think it's best to keep them all consistent.

Also: `git grep '\bit\b.* to_str'` does show a lot of places where we should probably switch to `#to_str`, but that is out of scope for this change.